### ISSUE TITLE
Incorrect version number

### ISSUE THREAD

**Issue by RomainQuidet** [Project-OSRM/osrm-backend#3893]
Hi,
I'm building from sources, release 5.6.0 on OSX (but brew has version 5.6.5 and shows the same issue):

in build directory, cmake ../ build an incorrect version.h file. 
The CMakeLists.txt file has not been updated since 5.5.0 and all tools are reporting this version. 

It's a big issue if you use osrm tool from 5.6 branch as routes prepared will fails to load.

**Comment by daniel-j-h** [comment]
Yep the 5.6 branch still has the 5.5 version number
https://github.com/Project-OSRM/osrm-backend/blob/5.6/CMakeLists.txt#L53-L55

We noticed that only after we already made a release. And now we can not increment the version number on the 5.6 branch, since this would break all consumers upgrading from a 5.6.x version:

https://github.com/Project-OSRM/osrm-backend/pull/3796

If you build osrm from source anyway, feel free to upgrade the version number.

Sorry, this was an oversight and should have never happened in the first place.
